axis.go

```diff
@@ -451,41 +451,70 @@ func (ts ConstantTicks) Ticks(float64, float64) []Tick {
 	return ts
 }
 
-// UnixTimeTicks is suitable for axes representing time values.
-// UnixTimeTicks expects values in Unix time seconds.
-type UnixTimeTicks struct {
+// TimeTicks is suitable for axes representing time values.
+// By default, TimeTicks expects values in (UTC) Unix time seconds.
+type TimeTicks struct {
 	// Ticker is used to generate a set of ticks.
 	// If nil, DefaultTicks will be used.
 	Ticker Ticker
 
 	// Format is the textual representation of the time value.
 	// If empty, time.RFC3339 will be used
 	Format string
+
+	// Converter takes a float64 value and converts it into a time.Time.
+	// If nil, UnixTimeConverter will be used.
+	Converter TimeFloatConverter
 }
 
-var _ Ticker = UnixTimeTicks{}
+var _ Ticker = TimeTicks{}
 
 // Ticks implements plot.Ticker.
-func (utt UnixTimeTicks) Ticks(min, max float64) []Tick {
-	if utt.Ticker == nil {
-		utt.Ticker = DefaultTicks{}
+func (tt TimeTicks) Ticks(min, max float64) []Tick {
+	if tt.Ticker == nil {
+		tt.Ticker = DefaultTicks{}
+	}
+	if tt.Format == "" {
+		tt.Format = time.RFC3339
 	}
-	if utt.Format == "" {
-		utt.Format = time.RFC3339
+	if tt.Converter == nil {
+		tt.Converter = UnixTimeConverter{}
 	}
 
-	ticks := utt.Ticker.Ticks(min, max)
+	ticks := tt.Ticker.Ticks(min, max)
 	for i := range ticks {
 		tick := &ticks[i]
 		if tick.Label == "" {
 			continue
 		}
-		t := time.Unix(int64(tick.Value), 0)
-		tick.Label = t.Format(utt.Format)
+		t := tt.Converter.UnmarshalTime(tick.Value)
+		tick.Label = t.Format(tt.Format)
 	}
 	return ticks
 }
 
+// TimeFloatConverter converts back and forth a time.Time value into
+// a float64.
+type TimeFloatConverter interface {
```

Comment:
Instead of using interface I would use 

    type TimeConverterFunc func(t float64) time.Time

and have

    func UnixTimeIn(loc *time.Location) TimeConverterFunc {
        return func(t float64) time.Time {
            return time.Unix(int64(t), 0).In(loc)
        }
    }

    var UTCUnixTime = UnixTimeIn(time.UTC) // use this as default converter